SimTKcommon/src/recorder.cpp

```diff
@@ -69,7 +69,7 @@ Recorder& Recorder::operator = ( const Recorder& r) {
 }
 
 void Recorder::operator>>=(double& value) {
-  if (!is_symbol()) throw std::runtime_error("Needs to be symbolic");
+  //if (!is_symbol()) throw std::runtime_error("Needs to be symbolic");
```

Comment:
Why do you need to comment this out? It seems that it might interact with Recorder.